compiler/damlc/tests/daml-test-files/Numeric.daml

```diff
@@ -10,3 +10,7 @@ import DA.Numeric
 testShift = script do
   shift @1 @2 1.0 === 10.0
   shift @2 @1 1.0 === 0.1
+
+
+testRound = script do
```

Comment:
I think you forgot to check in some code here

Reply:
I move it to `PreludeTest` but forget to remove this line. 
Thanks.